README.md

```
@@ -143,38 +143,46 @@ All environment variables now use a standardized `WANDERER_` prefix.
+   - `WANDERER_FEATURE_TRACK_KSPACE`: Track K-Space systems in addition to wormholes (default: true)
+
+   > **Note:** The following variables have been deprecated and removed:
+   >
+   > - `WANDERER_CHARACTER_TRACKING_ENABLED` (use `WANDERER_CHARACTER_NOTIFICATIONS_ENABLED` instead)
+   > - `WANDERER_SYSTEM_TRACKING_ENABLED` (use `WANDERER_SYSTEM_NOTIFICATIONS_ENABLED` instead)
 
 6. **Character Configuration**
 
    - `WANDERER_CHARACTER_EXCLUDE_LIST`: Comma-separated list of character IDs to exclude from tracking
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)